cfgov/v1/preprocessed/css/molecules/image-text-50-50.less

```diff
@@ -0,0 +1,48 @@
+/* ==========================================================================
+   cfgov-refresh
+   Image and Text 50-50 molecule
+   ========================================================================== */
+
+.m-image-text-50-50 {
+    position: relative;
+    padding-bottom: 60px;
```

Comment:
That seems like over using it. There's no reason to use a mixin when `30px / @font-size` or something similar would work fine all on it's own.